CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,37 @@
+
+## General Expectations
+
+### PR Discussion
+
+Comments on Pull Requests should remain relevant to the PR in question and not derail discussions.
+
+Under no circumstances are users to be attacked for their ideas or contributions. While constructive criticism is encouraged, toxicity or general mean-spirited behaviour will not be tolerated. All participants on a given PR or issue are expected to be civil. Failure to do so will result in disciplinary action.
+
+"Merge-nagging" or similar behaviour is not acceptable. Comments of this nature will result in warnings or an outright ban from the repository depending on general behaviour.
+
+If you exhibit behaviour that's considered to be a net-negative to the community (offensive commentary, repeat violations, constant nagging, personal attacks, etc.), you may be banned from other Paradise services (Discord, forums, server, wiki, etc.)
+
+Maintainers reserve the right to permanently revoke access from the repository if your behaviour is considered to be a net negative.
+
+### PR Approval/Objection Info
+
+Heads of Staff and Maintainers have the final say on Pull Requests. While thumbsup/thumbsdown reaction ratios are generally taken into account, they do not dictate whether or not a PR will be merged.
+
+After a twenty four hour minimum waiting period, Pull Requests can be merged once they receive approval from both a Head of Staff and a Maintainer. An exception is made for refactors and fixes, which may be merged at a Maintainer's discretion with no waiting period.
+
+Heads of Staff and Maintainers are not obligated to publicly state their objections to a Pull Request. Attacking or berating either of these roles over an objection will not be tolerated. Additionally, whining over the closure of a PR, the existence of an objection, or similar behaviour, will not be tolerated.
+
+### PR Expectations
+
+All Pull Requests are expected to be tested prior to submission. If a submitted Pull Request fails to pass CI checks, the likelihood of it being merged will be significantly lower. If you can't take the time to compile/test your Pull Request, do not expect a warm reception.
+
+It is expected that contributors discuss larger design changes on the forums prior to coding a Pull Request. The amount of time spent on any given Pull Request is not relevant. Maintainers are not responsible for contributors wasting their time creating features nobody asked for.
+
+Barring highly specific circumstances (such as single line changes, submissions from advanced users, or changes to repo documentation), we will not accept Pull Requests utilising the web editor.
+
+Pull Requests regarding heavy-handed nerfs, particularly immediately after said mechanic was used, will be tagged with `I ded pls nerf`. A bad experience with a particular mechanic is not a justification for nerfing it.
```

Comment:
I suggest re-wording this to: "Don't put up a PR to nerf a mechanic just after you die to it. If you do, it will be given this tag and likely not pass."
That way, the intent is clear, to discourage people from creating PRs nerfing something that they died to.
As written, this guideline arguably might be interpreted to restrict way more then that. It might be argued, for example, to apply even to PRs that someone has a "bad experience" with as a ghost (ie: as a ghost they watched a mechanic that seemed to be totally OP). Even though that (hopefully) is not the intent.

Reply:
It's intended to include salty ghosts/deadchat players, not just the person who died to it. Immediately issuing a revert to a mechanic the first time it's used is generally bad form. There are usually better ways to go about changing potentially troublesome mechanics than nerfing them into the floor, especially when a contributor hasn't allowed for any time regarding discussion (IE: immediately after a round).

Reply:
Having edited the original, this is the intent.

Ghosts/deadchat/etc. are expected to allow for time for discussion. Immediately posting nerfs the moment you see something doesn't allow for this. Encouraging people to post nerfs the second they think something is unfair is precisely what this is intending to stop.